### DISCUSSION TITLE
Hi，Do you have any recommended tools or ways for labeling own key point dataset?

### DISCUSSION THREAD

**Discussion by huangzhenjie** [open-mmlab/mmpose#615] [General]

**Comment by jin-s13** [comment]
Maybe you can try this [repo](https://github.com/Jeff-sjtu/labelKeypoint).

**Comment by huangzhenjie** [comment]
thanks，i  will try.

**Comment by jin-s13** [comment]
This tool also looks good. https://github.com/jsbroks/coco-annotator

**Comment by jin-s13** [comment]
Openmmlab have released a labeling tool. Check https://github.com/open-mmlab/labelbee-client
